MapleServer2/PacketHandlers/Game/UserSyncHandler.cs

```diff
@@ -39,9 +50,33 @@ public override void Handle(GameSession session, PacketReader packet)
 
             Packet syncPacket = SyncStatePacket.UserSync(session.FieldPlayer, syncStates);
             session.FieldManager.BroadcastPacket(syncPacket, session);
+            bool is200UnitsAway = (session.Player.SafeCoord - syncStates[0].Coord.ToFloat()).Length() > 200;
+            if (is200UnitsAway && session.FieldPlayer.Coord.Z == syncStates[0].Coord.ToFloat().Z && !session.Player.OnAirMount) // Save last coord if player is not falling
```

Comment:
Can this be a separate method that returns a boolean?

Reply:
sure